client_src/js/settings.js

```diff
@@ -426,50 +426,64 @@ const optionsCPU = {
 };
 const systemChartCPU = new ApexCharts(
   document.querySelector("#systemChartCPU"),
-  optionsCPU
+  optionsCPU,
 );
 systemChartCPU.render();
 const systemChartMemory = new ApexCharts(
   document.querySelector("#systemChartMemory"),
-  optionsMemory
+  optionsMemory,
 );
 systemChartMemory.render();
 setInterval(async function updateStatus() {
   let systemInfo = await Client.get("settings/sysInfo");
   systemInfo = await systemInfo.json();
 
-  if (
-    Object.keys(systemInfo).length === 0 &&
-    systemInfo.constructor === Object
```

Comment:
This made no sense to me, also given the fact that the if body was empty. Removed.